src/components/Question.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Question.propTypes = {
+  question: PropTypes.string.isRequired,
+  options: PropTypes.arrayOf(PropTypes.string).isRequired,
+  currentStep: PropTypes.number.isRequired,
+  totalSteps: PropTypes.number.isRequired,
+  steps: PropTypes.number.isRequired,
+  onUpdateSteps: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
+}
+
+export default function Question({
+  question,
+  options,
+  currentStep,
+  totalSteps,
+  steps,
+  onUpdateSteps,
+  onChange,
```

Comment:
`currentStep`, `totalSteps` and `steps` are all realted to displaying the current question, why should the `Question` component care about other `Question` components? Imho you can make your life easier by handling the switching logic at the same level where the `questions` state lives (and then put it all into a custom hook :) )

Reply:
Noooice! Changed it! Will work on the custom hook next!